src/main/java/org/folio/rest/service/action/validation/ActionValidationResult.java

```diff
@@ -9,6 +11,11 @@ public ActionValidationResult(String remainingAmount, String formattedAmount) {
     this.formattedAmount = formattedAmount;
   }
 
+  public ActionValidationResult(MonetaryValue remainingAmount, MonetaryValue formattedAmount) {
+    this.remainingAmount = remainingAmount.toString();
+    this.formattedAmount = formattedAmount.toString();
```

Comment:
I'm not sure about the name, makes it look like this variable is about formatting. Will `actionableAmount` work (as a generic version of `refundableAmount`)?

Reply:
I think `requestedAmount` is fine, it looks like we're using it when this object is created - `new ActionValidationResult(remainingBalance, requestedAmount)` so it would make sense if these names matched